recipes/sxiv

```diff
@@ -1,3 +1,4 @@
 (sxiv
- :fetcher gitlab
- :repo "contrapunctus/sxiv.el")
+ :fetcher git
+ :branch "dev"
```

Comment:
@contrapunctus-1 That branch does not exist. Did you forget to push it?

Reply:
Ah, whoops. Indeed, hadn't pushed it. Thanks for the heads up 😅

Reply:
This is why we want to build off the _default_ branch. I strongly suggest just using `master` or `main` for development.

chronometrist-goal is also broken because of a missing dev branch, I am updating the recipe to use the default branch instead so that the MELPA build passes.

Reply:
(See d5c27f29)